CheckSortedOrNotTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+
+/**
+ *
+ * @author goku
+ */
+import static org.junit.Assert.assertEquals;
+
+
+import org.junit.Test;
+public class CheckSortedOrNotTest {
+	//happy path
+	@Test
+	  public void checkSortedOrNotTest1() {
+	  CheckSortedOrNot check = new CheckSortedOrNot();
```

Comment:
Instead of creating new instance each time, new CheckSortedOrNot() can be initialized in @BeforeClass method